packages/create-block/lib/templates/esnext/src/style.scss.mustache

```diff
@@ -6,7 +6,7 @@
  */
 
 .wp-block-{{namespace}}-{{slug}} {
-	background-color: #000;
-	color: #fff;
+	background-color: theme(button);
```

Comment:
where is the `theme(button)` being defined / where is it coming from?

Reply:
It's from:
https://github.com/WordPress/gutenberg/tree/master/packages/postcss-themes

@youknowriad – is there any documentation we could link here in the code comment?

Reply:
Okay I'm curious, I just took a look at it and where is the postcss plugin the theme options from? 🤔 This could be huge 🤯

Reply:
Gutenberg uses it in a few places.

@jasmussen or @mtias – is it something that should be promoted or does it exist for backward compatibility? I need now sanity check :)

Reply:
I believe that allowing the admin, or even the environment that includes the block editor, to specify theme colors is desirable even going forward. This is not just about wp-admin offering color schemes, in my book, but about environments that include the block editor wanting to brand it a little bit. 

It might need a little more work, though. Take https://github.com/WordPress/gutenberg/pull/20460 as an example — this bakes in a baseline blue color that would be offered if someone that isn't wp-admin included the block editor. But it would be nice if that 3rd party were able to specify a color of their choice, not that particular blue I baked in there.

Reply:
Nice, thank you for clarifications. It also reminded me that #20460 got stuck 😞 What's more interesting is that we discussed with @fabiankaegy that we should allow overriding PostCSS config for `@wordpress/scripts` consumers. It makes me wonder if that would solve the issue you have. In other words, it would use different color schemas for WordPress core, for the Gutenberg plugin or standalone use case.

Reply:
> It also reminded me that #20460 got stuck 😞

To be fair, we might _not_ want to use that approach after all. I think that the idea of allowing _the environment that includes the block editor to provide the color_ might actually be better than _hardcoding that color in_, which that PR does.

So yes, if we can override PostCSS to inject a few colors, that might be much better. 

To be clear — I'm not suggesting that every single color should be customized, we'd still want to limit it to a moderated few spot colors for the UI, perhaps just starting with that single spot color that's currently blue. But if things work well, I could see that be carefully expanded!